index.html

```diff
@@ -49,5 +52,170 @@ <h3>I am a Software Developer</h3>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+
+		<div class="container">
+			<div class="project2">
+				<h2>Profesional Art Printing Data</h2>
+				<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				   <ul class="lang">
+					<li><a href="">html</a></li>
+					<li><a href="">bootstrap</a></li>
+					<li><a href="">Ruby</a></li>
+				</ul>
+				<button class="btn2"><a href="">See Project</a></button>
+			</div>
+		</div>
+	</section>
+
+	<section class="about-me">
+		<div class="container2">
+			<h1>About Me</h1>
+			<p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and 
+			   experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			   <button class="btn">Get My Resume</button>
+			   <img src="images/about img.png" alt="Vector shapes">
+		</div>
+
+		<div class="tools">
+			<h3><i class="fa fa-heart" aria-hidden="true"></i>Language</h3>
+			<ul>
+				<li>Javascript</li>
+				<li>Ruby</li>
+				<li>Html</li>
+				<li>Css</li>
+			</ul>
+		</div>
+
+		<div class="tools">
+			<h3><i class="fa fa-square" aria-hidden="true"></i>Frameworks</h3>
+			<ul>
```

Comment:
I am really impressed with your design sense, but actually, the purpose of this project is to create a web design that is pixel-perfect as per the given Figma Design. So kindly take a look at the Figma design again and try to redesign the section.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219882214-cd7c1769-78f5-4a71-ba7e-8d3d46e5e2ce.png)| ![image](https://user-images.githubusercontent.com/61361037/219882208-e5116178-5b49-4528-a018-4d0c7f51d59a.png)|